Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -75,6 +77,25 @@ async function run() {
         endPoint["subscriptionId"] = tl.getEndpointDataParameter(connectedServiceName, 'subscriptionid', true);
         endPoint["envAuthUrl"] = tl.getEndpointDataParameter(connectedServiceName, 'environmentAuthorityUrl', true);
         endPoint["url"] = tl.getEndpointUrl(connectedServiceName, true);
+        endPoint["environment"] = tl.getEndpointDataParameter(connectedServiceName, 'environment', true);
```

Comment:
Can we have a separate utility for endpoint? 
so that we can reduce the size of many many files(say azurermwebappseployment, azureappservicemanage)